Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM bitnami/node:8
+# We use the bitnami so that we can install 
+# both libsecret-1-dev libglib2.0-dev
+# 
```

Comment:
This is part of the why, as I accidentally did `node:8.11.3` without the bitnami fork and found it failing there, don't know why we need those though.

Reply:
The `bitnami` images are stripped down and offer things like `install_packages` (used below) that avoid crufting up the image with lots of dependencies. Some work went into optimizing this a while back, so I'd stick with it if possible.

Reply:
Perfect, I've added your summary to the Dockerfile 👍